CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ set(FlatBuffers_Library_SRCS
   include/flatbuffers/minireflect.h
   src/code_generators.cpp
   src/idl_parser.cpp
+  src/idl_gen_general.cpp
```

Comment:
sorry, did not catch this earlier. This is the Java/C# generator, which shouldn't be in the library.

Reply:
Ah I see. This was pulled in only for MakeCamel(), happy if I move that into util.cpp?